pihole-google.txt

```diff
@@ -972,7 +980,6 @@ pixel.google.com
 pki.google.com
 places.google.com
 play.google.com
-play.googleapis.com
```

Comment:
Please try not to delete lines and  add them somewhere else, it's almost impossible to review the additions like this because it looks like you have deleted them.

Reply:
good afternoon: I apologize for that, but I imagine that this would happen to me when I transfer the connections that I am detecting and others. And after adding them, I check for duplicates. I'll try to be more careful. Sorry for the inconveniences.